toolkit upgrade path (#2272)
timescale/docs: self-hosted/upgrades/about-upgrades.md

<PlanUpgrade />

<Highlight type="note">
If you use the Timescale Toolkit, ensure the `timescale_toolkit` extension is on

> version 1.4, then upgrade the `timescaledb` extension. If required, you

Reviewer: Since 1.16, we require at least 1.6.0 to upgrade now:
```suggestion
version 1.6.0, then upgrade the `timescaledb` extension. If required, you
```

> If you use the Timescale Toolkit, ensure the `timescale_toolkit` extension is on
> version 1.6.0, then upgrade the `timescaledb` extension. If required, you
> can then later upgrade the `timescale_toolkit` extension to the most

Reviewer 2: Not sure if we say Timescaledb toolkit or Timescale toolkit but the extension name has the *db* suffix.

```suggestion
If you use the Timescale Toolkit, ensure the `timescaledb_toolkit` extension is on
version 1.6.0, then upgrade the `timescaledb` extension. If required, you
can then later upgrade the `timescaledb_toolkit` extension to the most
```

> version 1.4 or later, then upgrade the `timescaledb` extension. If required, you
> can then upgrade the `timescale_toolkit` extension again to get to the most
> recent version.

Reviewer 3: ```suggestion
version 1.4, then upgrade the `timescaledb` extension. If required, you
can then later upgrade the `timescale_toolkit` extension to the most
recent version.
```
